bayesian_cnn_prometheus/analysis/correlation_analyzer.py

```diff
@@ -0,0 +1,159 @@
+import json
+import numpy as np
+
+from bayesian_cnn_prometheus.evaluation.utils import load_nifti_file # refactor idea: move this function to more 'general' directory
+from skimage import metrics
```

Comment:
Może "from skimage import metrics as sk_metrics" ? Później jak używasz "metrics" to tak wizualnie miesza się z self.metrics